### ISSUE TITLE
Usage with yarn workspaces / monorepo

### ISSUE THREAD

**Issue by mjbates7** [snipsnapdev/snipsnap-vscode#18]
Hello, I have the following directory structure (which is essentially a monorepo) but it doesn't pick up the snippets for my applications which live within `packages/`. I am using the [yarn workspaces feature](https://classic.yarnpkg.com/en/docs/workspaces). In my snipsnap.code-snippets file I get all the "base" snippets, but none targeted towards my applications. Running on Windows.

```
$ workspace
- .vscode
---- snipsnap.code-snippets
- node_modules
- packages
---- app1 (create react app)
-------- package.json
-------- tsconfig.json
-------- src
---- app2
---- appN
- package.json
- yarn.lock
```
Does it need to look further up the tree to pick up the yarn.lock file or any other ideas?

Debug from developer tools:
```
yarn.lock packages: []
package-lock packages: { dependencies: [] }
```

Thanks 😀

**Comment by mjbates7** [comment]
Opened a PR for this #20

**Comment by BolotskyDev** [comment]
Resolved in #20
